README.md

```diff
@@ -160,4 +160,13 @@ Computer configuration:
 + Processor - Intel® Core™ i5-6200U CPU @ 2.30GHz × 4 
 + OS type - 64-bit
 
+## Start instructions
+
+To start Mongoose for testing Kafka you need:
+
++ Take the last mongoose-base-x.x.xx.jar ([http://repo1.maven.org/maven2/com/github/emc-mongoose/mongoose-base/](http://repo1.maven.org/maven2/com/github/emc-mongoose/mongoose-base/))
++ Take the last preempt dependency ([http://repo1.maven.org/maven2/com/github/emc-mongoose/mongoose-storage-driver-preempt/](http://repo1.maven.org/maven2/com/github/emc-mongoose/mongoose-storage-driver-preempt/)) for Kafka storage driver and put in .mongoose\version_of_mongoose\ext
++ Build Kafka storage driver and put in .mongoose\version_of_mongoose\ext
```

Comment:
It's important that `.mongoose` directory is in the home directory. I usually refer this as `~/.mongoose`.